ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -30,6 +30,22 @@ object Main extends StrictLogging {
     val StartupError = 101
   }
 
+  object Logging {
+
+    import ch.qos.logback.classic.{Level => LogLevel}
+    import org.slf4j.{Logger, LoggerFactory}
+
+    def setGlobalLogLevel(verbosity: LogLevel): Unit = {
```

Comment:
This also appears to be duplicated with the corresponding [Sandbox code](https://github.com/digital-asset/daml/blob/main/ledger/sandbox-common/src/main/scala/platform/sandbox/logging/GlobalLogLevel.scala). Fine to keep it but please add a TODO.

Reply:
Todo added